docs/src/installation.md

```diff
@@ -0,0 +1,153 @@
+---
+description: How to install and run Chisel
+---
+
+# Installation
+
+Chisel is a [Scala](https://www.scala-lang.org/) library and compiler plugin.
+
+## Quickstart with Scala CLI
+
+The easiest way to install Chisel is to [install Scala CLI](https://scala-cli.virtuslab.org/install) to build and run the Chisel template:
+
+```bash
+wget https://github.com/chipsalliance/chisel/releases/latest/download/chisel-template.scala
+scala-cli chisel-template.scala
+```
+
+The Chisel template is a simple, single-file example of Chisel that emits Verilog to the screen.
+
+:::tip
+
+While more complex projects often use a build tool like SBT or Mill as described below,
+we still highly recommend using Scala CLI with the template for experimentation and
+writing small snippets to share with others.
+
+:::
+
+## Dependencies
+
+As described above, Scala CLI is a great "batteries included" way to use Chisel.
+It will automatically download and manage all dependencies of Chisel _including the JVM_.
+More complex projects will require the user to install the JDK and a build tool.
+
+:::info
+
+Note that each of these dependencies are projects with their own installation instructions.
+Please treat the commands below as suggestions and not directives.
+
+:::
+
+### Java Development Kit (JDK)
+
+Scala runs on the Java Virtual Machine (JVM), so it is necessary to install a JDK to use Chisel.
+Chisel works on any version of Java newer than Java; however, we recommend using an LTS release version 17 or newer.
+Note that Scala CLI requires Java 17 or newer so unless your system installation of Java is at least version 17, Scala CLI will download Java 17 for its own use.
```

Comment:
Maybe recommend 21? To pave the path to panama-binding in the future.

Reply:
As discussed in dev, it seems that we should give tooling around Scala a bit more time to mature before we recommend Java 21, but of course we will at some point, especially as the panama-binding comes on line.